apps/passport-server/src/services/generic-issuance/pipelines/PODPipeline/utils/atomize.ts

```diff
@@ -46,14 +51,26 @@ export function atomize(
       const cell = column.getValue(row);
       entries[key] = converter(cell);
     } else if (source.type === "configured") {
-      entries[key] = {
-        // @todo non-string configured values
-        type: "string",
-        value: source.value
-      };
+      if (entry.type === "string") {
+        entries[key] = {
+          type: "string",
+          value: source.value
+        } satisfies PODValue;
+      } else if (entry.type === "int") {
+        entries[key] = {
+          type: "int",
+          value: BigInt(source.value)
+        } satisfies PODValue;
+      } else if (entry.type === "cryptographic") {
+        entries[key] = {
+          type: "cryptographic",
+          value: BigInt(source.value)
+        } satisfies PODValue;
+      }
     } else if (
       source.type === "credentialSemaphoreID" ||
-      source.type === "credentialEmail"
+      source.type === "credentialEmail" ||
+      source.type === "signerPublicKey"
```

Comment:
Is this specifically this pipeline's signer's public key, or is it any public key (as a value)?
Looks like the former, though I don't think you generally need to put into a POD entry since it's already a part of the POD itself.

Reply:
Yes, it is unnecessary since the POD already includes it and there's a pseudo-entry for GPCs. I was still thinking of an earlier discussion in which it might have been necessary to add the public key explicitly as an entry, but there is no reason to do this now.